charset_normalizer/__init__.py

```diff
@@ -45,3 +47,26 @@
     "__version__",
     "VERSION",
 )
+
+
+def set_logging_handler(
```

Comment:
I would rather see that function in `utils.py` to avoid self importing the package in `api.py`

Reply:
And then exposing it publicly here